application/src/main/java/org/togetherjava/tjbot/commands/basic/VcActivityCommand.java

```diff
@@ -169,34 +169,45 @@ public void onSlashCommand(@NotNull SlashCommandEvent event) {
 
 
         String applicationId;
+        String applicationName;
 
         if (applicationOption != null) {
-            applicationId = VC_APPLICATION_TO_ID.get(applicationOption.getAsString());
+            applicationName = applicationOption.getAsString();
+            applicationId = VC_APPLICATION_TO_ID.get(applicationName);
         } else {
             applicationId = idOption.getAsString();
+            applicationName = applicationId;
+
+            // Get the application name from the ID
+            for (var entry : VC_APPLICATION_TO_ID.entrySet()) {
+                if (applicationId.equals(entry.getValue())) {
+                    applicationName = entry.getKey();
+                    break;
```

Comment:
* What if its never found? What behavior do you want then?
* Might create a little helper method `private static Optional<K> <K, V> getKeyByValue(Map<K, V> map, V value)`, makes this more readable.

Reply:
> What if its never found? What behavior do you want then?

If the activity is not found the bot currently replies `Something went wrong :/` and logs a warning. Should it do something else?

Reply:
He's talking about when the name isn't found, not when the activity doesn't exist.

And when the name isn't found imo it should say 

`Tais#0002 wants to start an activity.` 
Instead of
`Tais#0002 wants to start 01258284129.`

Reply:
Oh yup ok, thanks :)

Reply:
> What if its never found? What behavior do you want then?

Changed in 16db30d18f0dbe01b7948578f5f9ad451f41c605

> Might create a little helper method private static Optional<K> <K, V> getKeyByValue(Map<K, V> map, V value), makes this more readable.

Changed in 107d309b978ad9aeb2783e0fb415b1ca750841e1